documentation/how-to/print-to-pdf.en.md

```diff
@@ -36,3 +36,9 @@ Feature(s) to PDF*.
 
 It is possible to print a single atlas feature through the feature form by clicking on the
 *Menu ( ⁝ )* button and selecting the *Print Atlas Feature to PDF*.
+
+## Synchronize Generated Layouts
+
+To sync layout PDFs with QFieldCloud, add the **'layouts'** folder to the list of *Attachment directories* in the *Project Properties* panel under the *QField* section. All generated layout PDFs will be stored in this folder.
```

Comment:
```suggestion
To sync layout PDFs with QFieldCloud, add the **'layouts'** folder to the list of *Attachment directories* in the *Project Properties* panel under the *QField* section.
All generated layout PDFs will be stored in this folder.
```

Please do one sentence per line, makes it easier to manage in the future.

I am afraid the first sentence is not correct, or at least misleading. This will only make the `layouts` dir downloaded to QFieldCloud. @nirvn do I miss something?